packages/features/portfolio/src/hooks/use-vault-holdings-table.tsx

```diff
@@ -85,12 +86,12 @@ export function getVaultReinvestmentDates(
       ? reinvestmentData[vaultAddress]
       : undefined;
   if (vaultReinvestmentData && vaultReinvestmentData.length > 0) {
-    arbReinvestmentDate = moment(vaultReinvestmentData[0].timestamp * 1000)
-      .add(1, 'days')
-      .format('MMM DD YYYY');
-    mainnetReinvestmentDate = moment(vaultReinvestmentData[0].timestamp * 1000)
-      .add(7, 'days')
-      .format('MMM DD YYYY');
+    arbReinvestmentDate = getDateString(vaultReinvestmentData[0].timestamp);
+    mainnetReinvestmentDate = getDateString(
```

Comment:
Why are the two getDateString calls different?

Reply:
I pushed a fix this for this. Lmk if it's what you're expecting